CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.6.2] - 2020-04-17
+
+* Clean up code.
```

Comment:
We should probably have mentioned that is a breaking change, clients that previously relied on `GoogleFonts.config.allowHttp` now need to change to `GoogleFonts.config.allowRuntimeFetching`.

Reply:
Never mind, I see that this was done in previous verison.